vscode-extension/src/view/app/registry/ReferenceWidgetDocumentTransform.ts

```diff
@@ -0,0 +1,244 @@
+/*******************************************************************************
```

Comment:
This file should probably be exported from its original location instead of being copied

Reply:
done